packages/backend/package.json

```diff
@@ -25,10 +25,10 @@
     "@backstage/plugin-auth-backend": "0.18.8",
     "@backstage/plugin-auth-node": "0.2.19",
     "@backstage/plugin-azure-devops-backend": "0.3.29",
-    "@backstage/plugin-catalog-backend": "1.12.4",
-    "@backstage/plugin-catalog-backend-module-github": "0.3.7",
-    "@backstage/plugin-catalog-backend-module-gitlab": "0.2.6",
-    "@backstage/plugin-catalog-backend-module-openapi": "0.1.18",
+    "@backstage/plugin-catalog-backend": "^1.13.0-next",
```

Comment:
@davidfestal are you preparing for the 1.18.0 release? Currently our policy is not to use prereleases. 

cc @christophe-f

Reply:
In fact I need those, because the backend-plugin-manager is not compatible with previous releases.
I relied on the expectation that finally the showcase application, will depend on the 1.18.0 release.

Reply:
Now that the 1.18.0 release has been issued, I updated and fixed the version of those dependencies to the 1.18.0 ones.
Is it OK for you @schultzp2020